parsl/serialize/concretes.py

```diff
@@ -17,7 +17,7 @@ class PickleSerializer(SerializerBase):
     * closures, generators and coroutines
     """
 
-    _identifier = b'01'
+    identifier = b'01'
 
```

Comment:
If we've defined the property in the `SerializerBase`, what's the need for these class properties in the subclasses anymore?

Reply:
some wire compatiblity (though I am not particularly fussed there) and some serialised object length: (47 bytes header vs 2 bytes header)

Reply:
*nod; so the base class property is meant as a _fallback_, then?  I dig it.

Reply:
in general a user defined serialiser *should* use that base class impl and not define their own identifier: if you define a different identifier explicitly which does not fit the importable/locatable syntax, you also take on responsibility for ensuring anyone who might deserialise also knows what to do with this identifier. So I guess I'm imagining that only being used if someone really wants to hard code identifiers and is modifying the parsl source code.